orangewidget/utils/signals.py

```diff
@@ -159,10 +161,10 @@ def set_train_data(self, data):
 
     def __init__(self, name, type, id=None, doc=None, replaces=None, *,
                  multiple=False, default=False, explicit=False,
-                 auto_summary=True, closing_sentinel=None):
+                 auto_summary=None, closing_sentinel=None):
         flags = self.get_flags(multiple, default, explicit, False)
         super().__init__(name, type, "", flags, id, doc, replaces or [])
-        self.auto_summary = auto_summary and can_summarize(type, name)
+        self.auto_summary = can_summarize(type, name, auto_summary)
```

Comment:
The docstring for auto_summary should be updated.

Reply:
Updated. Plus added a docstring for `summarize`.